### PR TITLE
Add additional error codes

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: openprovider/data/exception_map.py [hunk 1]
```diff
@@ -265,6 +298,15 @@
...
     874: ValidationError,           # Template ID is empty.
+    875: RuleViolation,             # Not enough nameservers given
+    876: RuleViolation,             # It is not possible to use MX records that point to an IP address in the range 79.99.129.0/24.
+    890: RuleViolation,             # Incorrect DNSSEC record format.
+    891: RuleViolation,             # DNSSEC is not supported for this extension
+    892: RuleViolation,             # Protocol field of the dnskey record must be 3
+    893: RuleViolation,             # Algorithm field has wrong value
+    894: RuleViolation,             # Digest type field has wrong value
+    895: RuleViolation,             # DNSSEC keys and digests should be BASE64 encoded
+    896: RuleViolation,             # DNSSEC processing error
```
Comment (ekohl) [inline openprovider/data/exception_map.py]: Perhaps this should be a ServerError?

Comment (wandernauta) [inline openprovider/data/exception_map.py]: Depends on whether the user or OP caused it, I guess.

Comment (ekohl) [inline openprovider/data/exception_map.py]: I had that problem with a lot of these errors. Hard to find out just by the error message. Downgrade to the generic exception?

Comment (wandernauta) [inline openprovider/data/exception_map.py]: Either that, or ask OP for a clarification.

**Pull request merged**